### ISSUE TITLE
Update current test cases

### ISSUE THREAD

**Issue by kxepal** [thebuzzmedia/universal-binary-json#8]
subj.

**Comment by DbxDev** [comment]
Reviving this old issue. The test cases provided are clearly not following the current specifications. I would be good to move those files to a directory specifying the draft version they follow.
For example `TwitterTimeline.ubj` starts with byte 111 which is [o] (for object I guess), but current draft uses [{].

**Comment by rkalla** [comment]
You are right those examples are all Draft 8. I handed ownership to Steven
Braeger recently and I think he has some plans on cleaning a lot of this up.
